src/commands/handlers/messaging.js

```diff
@@ -70,8 +70,8 @@ var handlers = {
 
         if ((message.charAt(0) === '\01') && (message.charAt(message.length - 1) === '\01')) {
             // CTCP request
-            if (message.substr(1, 6) === 'ACTION') {
-
+            var ctcp_command = message.slice(1, -1).split(' ')[0];
```

Comment:
It doesn't look like it would matter. The [event docs](https://github.com/kiwiirc/irc-framework/blob/917341a58eec8bd8d821432df4a86440801d5f32/docs/events.md#L328) example `ctcp request` event uses all uppercase.